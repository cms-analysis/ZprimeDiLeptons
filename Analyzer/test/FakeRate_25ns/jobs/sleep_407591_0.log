000 (407591.000.000) 09/13 23:20:31 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (407591.000.000) 09/13 23:36:43 Job executing on host: <90.147.169.68:38397>
...
006 (407591.000.000) 09/13 23:36:51 Image size of job updated: 121188
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
005 (407591.000.000) 09/13 23:36:52 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:05, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:05, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98527  -  Run Bytes Sent By Job
	69219  -  Run Bytes Received By Job
	98527  -  Total Bytes Sent By Job
	69219  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :      214       75   2654512
	   Memory (MB)          :        0     2500      2500
...
