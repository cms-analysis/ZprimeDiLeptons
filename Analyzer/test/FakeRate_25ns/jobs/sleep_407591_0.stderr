mv: impossibile fare stat di `/home/condor/dir_17860/CMSSW803-Analyse_ZprimeToMuMu_13TeV.root': No such file or directory
mv: impossibile fare stat di `/home/condor/dir_17860/CMSSW803-Analyse_ZprimeToMuMu_13TeV_cand.txt': No such file or directory
